src/Security.php

```diff
@@ -55,4 +55,21 @@ public static function wp_redirect($url) {
     return $url;
   }
 
+  /**
+   * @implemements wp_headers
+   *
+   * @return array
+   */
+  public static function additional_securityheaders(array $headers):array {
+    if (!is_admin()) {
+      $headers['Referrer-Policy']           ??= defined('CORE_STANDARDS_Referrer_Policy') ? CORE_STANDARDS_Referrer_Policy : 'no-referrer-when-downgrade';
```

Comment:
1. Usage of the [null coalesce assignment operator](https://www.php.net/manual/en/language.operators.assignment.php#language.operators.assignment.other) is wrong here; the tested value must be variable that may be null. `defined()` is a language expression and never null but true or false.

2. PHP constant names must always be all-uppercase.

3. Can you sort the assignments alphabetically by header name, please?  It's not clear in which order they are currently.

4. Minor: The equal sign of the assignments on multiple lines should not be aligned. It causes an unnecessary line changes in future PRs.